src/Writer/index.js

```diff
@@ -20,21 +20,25 @@ const constant = x => () => x
 
 function _Writer(Monoid) {
   if(!isMonoid(Monoid)) {
-    throw new TypeError('Writer: Monoid required for construction')
+    throw new TypeError('Writer: Argument must be a Monoid Constructor')
   }
 
-  const _of =
-    x => Writer(Monoid.empty().valueOf(), x)
+  const _of = x =>
+    Writer(Monoid.empty(), x)
 
   const _type =
-    constant(`${__type}( ${Monoid.type()} )`)
+    constant(`${__type}(${Monoid.type()})`)
```

Comment:
Not required for review but I'm curious, why make this a func instead of a variable?

Reply:
Everything except for the sanctuary type is a function. Mostly to keep the API :corn:sistant. But it has the benefit of not accidentally overriding a value that we depend on for most of our type operations. If that makes sense?